### ISSUE TITLE
flux:brand breaks WCAG 2.1 "Non-text Content (Level A)" Criterion

### ISSUE THREAD

**Issue by lonebadger99** [livewire/flux#378]
When using flux:brand, the code generated has no alt for the image. Furthermore, if we supply an alt, the alt is applied to the ```<a>``` tag and not the ```<img>``` tag so it breaks the criterion for WCAG 2.1 "[Non-text Content (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html)"

```
<flux:brand href="#main" logo="https://fluxui.dev/img/demo/logo.png" name="name" alt="alternative text" class="max-lg:hidden dark:hidden" />
```

```
<a href="#main" class="h-10 flex items-center mr-4 gap-2 max-lg:hidden dark:hidden" alt="alternative text" data-flux-brand="">
        <div class="size-6 rounded overflow-hidden">
                            <img src="https://fluxui.dev/img/demo/logo.png">
                    </div>

        <div class="text-sm text-zinc-900 dark:text-zinc-100 font-medium truncate">name</div>
 </a>
```

**Comment by calebporzio** [comment]
Good catch thanks - I'm now forwarding alt to <img>

**Comment by lonebadger99** [comment]
This issue is still not resolved. Maybe the issue was wrongly closed or the code not included in release 10?
